README.md

```diff
@@ -66,4 +104,4 @@ Storybook reference links:
 - [Writing Stories](https://storybook.js.org/docs/basics/writing-stories/)
 - [Using Knobs](https://github.com/storybookjs/storybook/tree/master/addons/knobs)
 
-\* One note about using knobs is that you have to use the **`text`** method for defining `array` or `object` properties for lit-element. 
+\* One note about using knobs is that you have to use the **`text`** method for defining `array` or `object` properties for lit-element.
```

Comment:
Did we not end up getting lit-html working in Storybook?

Reply:
I believe we did - what's the question here? In case it helps, here is some background.

This comment is clarifying that knobs are, frankly, annoying to use, and I would argue do not allow for client-presentable code. We'll be discussing this more in the coming sprints.

Additionally, given that Storybook is intended for use as how the client would mount our components and the logic layer, we cannot assume they are using LitElement.